### DISCUSSION TITLE
Some questions about Demo official website

### DISCUSSION THREAD

**Discussion by muyu1285** [questdb/questdb#2876] [Ideas]
May I ask, in the Demo of the official website, does 1.6 billion data use a single node server or a multi-node server? If it is multi-node, what is the maximum data volume of a single node to ensure the speed of ms class

**Comment by puzpuzpuz** [comment]
Hi @muyu1285 the demo box is a single c5.metal AWS instance with a gp3 EBS volume. Exact configuration of a server that would guarantee ms latency depends on your data volume and load.

**Reply by muyu1285** [reply]
ok, thanks
@puzpuzpuz

**Comment by muyu1285** [comment]
@puzpuzpuz 
Excuse me, how to cluster, that is, multi-server deployment application!
Which section of the docs?
Ask for help！

**Reply by puzpuzpuz** [reply]
Hi @muyu1285,

We have replication in progress, but it's hard to share any ETA on its readiness. In the meanwhile, you could implement some workarounds like a TCP proxy that would be streaming ILP data to multiple QuestDB instances or application-level sharding.

**Reply by muyu1285** [reply]
ok, you guys are doing great.
